backend/apps/sandbox/views.py

```
@@ -302,110 +317,108 @@ class SnippetCollectionViewSet(viewsets.ModelViewSet):
             return Response(
-                {'error': 'Code is required'},
-                status=status.HTTP_400_BAD_REQUEST
+                {"error": "Code is required"}, status=status.HTTP_400_BAD_REQUEST
             )
-        
+
         try:
             import json
+
             payload = json.loads(payload_raw)
         except:
```

Ruff (0.15.21)

[error] Do not use bare `except`

(E722)